### DISCUSSION TITLE
Maintenance windows API deprecated (uses /settings/ now)

### DISCUSSION THREAD

**Discussion by ElodieLam** [Dynatrace/dynatrace-configuration-as-code#687] [Ideas]
Hi,

Previously used API to create Maintenance windows (/api/config/v1/maintenanceWindows) is now deprecated and is now /settings/ :

![image](https://user-images.githubusercontent.com/43603718/179792383-10a4bc83-b0f1-459d-88e8-7ec0fcd6e419.png)

![image](https://user-images.githubusercontent.com/43603718/179792510-aab5a308-bb25-4be2-a3b6-1ebff194a113.png)

This setting is highly valuable ! :)

Thank you !
Kind regards,
Elodie

**Comment by roger-mario** [comment]
I'm also interessted in getting configs from the new /settings/objects API - could any tell me how to integrate it?

![image](https://user-images.githubusercontent.com/103379642/185362628-ec7858ea-13ff-43cf-9f34-f2806d586aff.png)

**Reply by UnseenWizzard** [reply]
Hi @roger-mario. Support for the settings API is currently work in progress and will be part of an upcoming 2.0 release 
https://github.com/dynatrace-oss/dynatrace-monitoring-as-code/issues/107

As you mention this API is substantially different to the previous APIs so I can sadly not give you an ETA on this at the moment.
